.github/workflows/conbench_upload.yml

```diff
@@ -0,0 +1,104 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Upload Benchmark Results
+on:
+  workflow_dispatch:
+    inputs:
+      run_id:
+        description: 'workflow run id to use the artifacts from'
+        required: true
+  workflow_run:
+    workflows: ["Linux Benchmark"]
+    types:
+      - completed
```

Comment:
Nice. This whole workflow is very cool. Do you know why this workflow wasn't triggered by [this PR's successful benchmark run](https://github.com/facebookincubator/velox/actions/runs/4012873483/jobs/6891667952)?

Reply:
It has to be on the default branch to get triggered as it runs with elevated permissions.